### PR TITLE
View task by date

### LINKED ISSUE #45: View all the tasks on a particular date
As a busy student, I want to view all the tasks on a particular date so that I can easily plan my schedule.

### PR DESCRIPTION
**Pull request by khiaxeng**
Add command to view task by date
eg:
view 2021-03-24
view today
view tomorrow

These commands should list out all tasks with the matching dates.

### CODE REVIEW

**Review by noelmathewisaac — APPROVED** [review]

**Review by noelmathewisaac — APPROVED** [review]
LGTM

File: src/test/java/seedu/taskify/logic/parser/ViewCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        LocalDate todayDate = LocalDate.now();
+        ViewCommand expectedTodayViewCommand = new ViewCommand(new TaskHasSameDatePredicate(todayDate));
+        assertParseSuccess(parser, "today", expectedTodayViewCommand);
+
+        // using 'view tomorrow'
+        LocalDate tomorrowDate = LocalDate.now().plus(1, ChronoUnit.DAYS);
+        ViewCommand expectedTomorrowViewCommand = new ViewCommand(new TaskHasSameDatePredicate(tomorrowDate));
+        assertParseSuccess(parser, "tomorrow", expectedTomorrowViewCommand);
+    }
+}
```
Comment (CSmortal) [inline src/test/java/seedu/taskify/logic/parser/ViewCommandParserTest.java]: i think there should be a test for parsing with invalid args

Comment (khiaxeng) [reply src/test/java/seedu/taskify/logic/parser/ViewCommandParserTest.java]: Done, added a test case for invalid args

**Review by CSmortal — APPROVED** [review]
LGTM!

**Pull request merged**